django/cantusdb_project/main_app/templates/chant_detail.html

```diff
@@ -6,6 +6,16 @@
 <div class="container">
     <div class="row">
         <div class="mr-3 p-3 col-lg-8 bg-white rounded">
+
+            <!--Display "submit success" message -->
+            {% if messages %}
```

Comment:
Please add a level of indentation after the Django template tags `for` and `if`. It's not important whether we use indentation or not, but we need pick one and stay consistent.